code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -183,10 +183,13 @@
 		if (!M.stuttering) M.stuttering = 1
 		M.stuttering += 4
 		M.Dizzy(5)
+		if(iscultist(M) && prob(5))
+-			M.say(pick("Av'te Nar'sie","Pa'lid Mors","INO INO ORA ANA","SAT ANA!","Daim'niodeis Arc'iai Le'eones","R'ge Na'sie","Diabo us Vo'iscum","Eld' Mon Nobis"))
 	if(data >= 75 && prob(33))	// 30 units, 135 seconds
 		if (!M.confused) M.confused = 1
 		M.confused += 3
-		if((is_handofgod_cultist(M) && !is_handofgod_prophet(M)))
+		if(iscultist(M) || (is_handofgod_cultist(M) && !is_handofgod_prophet(M)))
+			ticker.mode.remove_cultist(M.mind)
```

Comment:
The deconversion takes more effort than almost any other. You have to babysit the cultists and keep them separated since a single loose cultist with a tome can undo everything. You also also have to babysit if you want to confirm that they've been deconverted since theres only a brief moment when you can tell that a cultist is purified. 

Also being able to summon cultists during this process makes it super risky as opposed to shooting them on the spot.